app/frontend/src/features/auth/email/ChangeEmail.tsx

```diff
@@ -60,7 +60,7 @@ export default function ChangeEmail(accountInfo: GetAccountInfoRes.AsObject) {
       <PageTitle>{CHANGE_EMAIL}</PageTitle>
       <>
         <Typography variant="body1">
-          {YOUR_EMAIL_IS} <pre>{accountInfo.email}</pre>
+          {YOUR_EMAIL_IS} <span>{accountInfo.email}</span>
```

Comment:
The console warning on this being invalid HTML nesting has been annoying me for some time, nice!